app/api/services/suppliers.py

```diff
@@ -16,11 +22,24 @@ class SuppliersService(Service):
     def __init__(self, *args, **kwargs):
         super(SuppliersService, self).__init__(*args, **kwargs)
 
-    def get_suppliers_by_contact_email(self, emails):
-        return (db.session.query(Supplier)
-                .filter(func.lower(Supplier.data['contact_email'].astext).in_(emails))
-                .filter(Supplier.status != 'deleted')
-                .all())
+    def get_suppliers_by_contact_email(self, emails, area_of_expertise, lot_id):
+        query = db.session.query(Supplier)
+
+        if lot_id == 9:
```

Comment:
We should make sure to reference lots by slug and not id e.g. `lots_service.find(slug='digital-professionals').one_or_none()`

Reply:
fixed